src/aggregators/direct.jl

```diff
@@ -58,16 +58,16 @@ end
 # calculate the next jump / jump time
 function generate_jumps!(p::DirectJumpAggregation, integrator, u, params, t)
     p.sum_rate, ttnj = time_to_next_jump(p, u, params, t)
-    @fastmath p.next_jump_time = t + ttnj
```

Comment:
@ChrisRackauckas I'll go through these for you. This is just an addition, so I don't see why we'd benefit from @fastmath (and `ttnj` could be `Inf` so I'm not sure it is reasonable to keep `fastmath`).

Reply:
Using `add_fast` here I think is fine though, since it wouldn't decrease the accuracy and NaN or Inf would both trigger the same exit behavior in the integrator